server/app.js

```diff
@@ -11,13 +11,21 @@ const logger = require("morgan");
 
 const authRouter = require("./routes/auth");
 const userRouter = require("./routes/user");
+const profileRouter = require("./routes/profile");
+
+require("dotenv").config();
```

Comment:
Could this be moved to the top of the file?

Reply:
Absolutely